modules/ROOT/pages/endpoints.adoc

```diff
@@ -5,11 +5,13 @@ endif::[]
 
 There are three types of endpoints within AnyPoint Partner Manager
 
-** Receive endpoints - On the Host side, for an inbound transmission you need endpoints your organization uses to receive transmissions from your partners. These endpoints are resources your organization exposes to your trading partners.
+** Receive endpoints - On the Host side, for an inbound transmission you need endpoints your organization uses to receive transmissions from your partners.
+These endpoints are resources your organization exposes to your trading partners.
```

Comment:
Is this sentence necessary? Seems implied by the previous.

Reply:
Leaving it for now.